### PR TITLE
[ID] Add "Full combo", "Combo multiple effect", "Slidertails"

### PR DESCRIPTION
**Pull request by mousesalt**
Add Indonesian [Full combo](https://osu.ppy.sh/wiki/en/Full_combo), [Combo multiple effect](https://osu.ppy.sh/wiki/en/Glossary/Combo_multiplier_effect), [Slidertails](https://osu.ppy.sh/wiki/en/Hit_object/Slidertail)

### CODE REVIEW

File: wiki/Glossary/Combo_multiplier_effect/id.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - score v1
+  - skor v1
+---
+
+# Efek pengganda kombo
+
+**Efek pengganda kombo** adalah sebuah bug di dalam [scoreV1](/wiki/Score/ScoreV1) yang mana [skor](/wiki/Score) diizinkan untuk dihitung secara mundur. Ini adalah sebuah kecacatan *integer* (bilangan bulat) 32-bit bertanda, di mana (dalam komputasi) *integer* memiliki nilai maksimal `2,147,483,647`. Setelah nilai batasan tersebut tercapai, skor akan mulai dihitung mundur. Di dalam [scoreV2](/wiki/Score#scorev2) masalah ini telah diperbaiki dengan membatasi skor pada 1 juta poin, bukan pengubah akuntansi.
```
Comment (Nivalyx) [inline wiki/Glossary/Combo_multiplier_effect/id.md:10]: the word "accounting" in the og article here doesn't literally translate to "akuntansi" orz D:

i'd say if any you can just delete the `,bukan pengubah akuntansi.` part of the sentence here

File: wiki/Full_combo/id.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - FC
+  - full kombo
```
Comment (Nivalyx) [inline wiki/Full_combo/id.md:5]: might leave every instance of "full combo" untranslated here tbh, genuinely speaking i have never seen anyone referring to fc as `full kombo` or `kombo penuh` or the likes of it

File: wiki/Full_combo/id.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+# Full combo
+
+**Full combo** (atau **Full kombo**, kependekannya adalah *FC*) merupakan istilah yang digunakan untuk menggambarkan pemain-pemain yang memperoleh kombo semaksimal mungkin pada suatu [beatmap](/wiki/Beatmap). Penilaian Full kombo adalah dengan menyelesaikan suatu beatmap tanpa memperoleh miss, [slider break](/wiki/Gameplay/Slider_break), maupun miss pada saat [akhir slider](/wiki/Hit_object/Slidertail).
```
Comment (Nivalyx) [inline wiki/Full_combo/id.md:10]: `**Full combo** (atau yang umum disingkat sebagai *FC*) merupakan istilah yang...`

File: wiki/Glossary/Combo_multiplier_effect/id.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Efek pengganda kombo terjadi pada [osu!standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), dan [osu!catch](/wiki/Game_mode/osu!catch) saja. Ini disebabkan karena mode permainan tersebut menggunakan [kombo](/wiki/Glossary/Combo_(score_multiplier)) dari arus pemain sebagai bagian dari perhitungan skor. Artinya seorang pemain akan mendapatkan skor yang lebih tinggi jika memperoleh [full combo (FC)](/wiki/Glossary#fc) daripada pemain lainnya yang memainkan beatmap yang sama dengan kombo yang pecah.
```
Comment (Nivalyx) [inline wiki/Glossary/Combo_multiplier_effect/id.md:12]: would suggest `dari arus pemain` to be deleted altogether for sentence cohesion

also the words `broken combo` here are actually something that was derived from the term `combobreak` and doesn't literally translate into `kombo yang pecah` or the likes of it; for the second sentence, you can consider sth like `Dengan kata lain, seorang pemain yang memperoleh [full combo (FC)](/wiki/Glossary#fc) akan mendapatkan skor yang lebih tinggi daripada pemain lainnya yang memainkan beatmap yang sama tanpa rangkaian kombo yang sempurna.`

File: wiki/Full_combo/id.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+# Full combo
+
+**Full combo** (atau yang umum disingkat sebagai *FC*) merupakan istilah yang digunakan untuk menggambarkan pemain-pemain yang memperoleh combo semaksimal mungkin pada suatu [beatmap](/wiki/Beatmap). Penilaian Full combo adalah dengan menyelesaikan suatu beatmap tanpa memperoleh miss, [slider break](/wiki/Gameplay/Slider_break), maupun miss pada saat [akhir slider](/wiki/Hit_object/Slidertail).
+
+Dampak skor karena kehilangan combo pada saat miss diakhir slider, secara luas dalam komunitas dianggap sebagai full combo. Anggapan ini berbeda dengan skor di dalam klien gim dan tampilan situs web.
+
+![Sebuah tangkapan layar dari full combo dan combo break](img/combo-comparison.png "Skor teratas adalah full combo dan skor terbawah adalah combo break.")
+
+Dikarenakan [efek pengganda combo](/wiki/Glossary/Combo_multiplier_effect), full combo akan memberikan [skor](/wiki/Score) paling banyak di dalam [mode permainan](/wiki/Game_mode) [osu!standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), dan [osu!catch](/wiki/Game_mode/osu!catch).
```
Comment (DoryanR) [inline wiki/Full_combo/id.md:15]:
```suggestion
Dikarenakan [efek pengganda combo](/wiki/Glossary/Combo_multiplier_effect), full combo akan memberikan [skor](/wiki/Score) paling banyak di dalam [mode permainan](/wiki/Game_mode) [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), dan [osu!catch](/wiki/Game_mode/osu!catch).
```

File: wiki/Glossary/Combo_multiplier_effect/id.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+  - score v1
+  - skor v1
+---
+
+# Efek pengganda kombo
+
+**Efek pengganda kombo** adalah sebuah bug di dalam [scoreV1](/wiki/Score/ScoreV1) yang mana [skor](/wiki/Score) diizinkan untuk dihitung secara mundur. Ini adalah sebuah kecacatan *integer* (bilangan bulat) 32-bit bertanda, di mana (dalam komputasi) *integer* memiliki nilai maksimal `2,147,483,647`. Setelah nilai batasan tersebut tercapai, skor akan mulai dihitung mundur. Di dalam [scoreV2](/wiki/Score#scorev2) masalah ini telah diperbaiki dengan membatasi skor pada 1 juta poin.
+
+Efek pengganda kombo terjadi pada [osu!standard](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), dan [osu!catch](/wiki/Game_mode/osu!catch) saja. Ini disebabkan karena mode permainan tersebut menggunakan [kombo](/wiki/Glossary/kombo_(score_multiplier)) dari sebagai bagian dari perhitungan skor. Dengan kata lain, seorang pemain yang memperoleh [full combo (FC)](/wiki/Glossary#fc) lebih tinggi daripada pemain lainnya pada beatmap yang sama akan mendapatkan skor yang lebih tinggi tanpa rangkaian kombo yang sempurna.
```
Comment (DoryanR) [inline wiki/Glossary/Combo_multiplier_effect/id.md:12]:
```suggestion
Efek pengganda kombo terjadi pada [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), dan [osu!catch](/wiki/Game_mode/osu!catch) saja. Ini disebabkan karena mode permainan tersebut menggunakan [kombo](/wiki/Glossary/kombo_(score_multiplier)) dari sebagai bagian dari perhitungan skor. Dengan kata lain, seorang pemain yang memperoleh [full combo (FC)](/wiki/Glossary#fc) lebih tinggi daripada pemain lainnya pada beatmap yang sama akan mendapatkan skor yang lebih tinggi tanpa rangkaian kombo yang sempurna.
```

File: wiki/Hit_object/Slidertail/id.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+stub: true
+tags:
+  - sliderend
+  - akhir slider
+---
+
+# Slidertail
+
+*Slidertail* atau **sliderend**, atau yang umum dirujuk sebagai *akhir slider*, merupakan suatu bagian dari [slider](/wiki/Hit_object/Slider) yang mengharuskan pemain untuk melepaskan tombol setelah mengikuti sebuah [sliderbody](/wiki/Hit_object/Sliderbody). Slidertail hanya bisa ditemukan di dalam [beatmap](/wiki/Beatmap) [osu!standard](/wiki/Game_mode/osu!). Slidertail bisa juga dilihat secara penuh sebagai sebuah circle atau tidak dapat dilihat sebagai bagian dari sliderbody, tergantung pada versi skin yang kamu gunakan. Warna dari slidertail ditentukan dari warna kombo yang digunakan oleh mapper pada beatmap mereka.
```
Comment (DoryanR) [inline wiki/Hit_object/Slidertail/id.md:10]:
```suggestion
*Slidertail* atau **sliderend**, atau yang umum dirujuk sebagai *akhir slider*, merupakan suatu bagian dari [slider](/wiki/Hit_object/Slider) yang mengharuskan pemain untuk melepaskan tombol setelah mengikuti sebuah [sliderbody](/wiki/Hit_object/Sliderbody). Slidertail hanya bisa ditemukan di dalam [beatmap](/wiki/Beatmap) [osu!](/wiki/Game_mode/osu!). Slidertail bisa juga dilihat secara penuh sebagai sebuah circle atau tidak dapat dilihat sebagai bagian dari sliderbody, tergantung pada versi skin yang kamu gunakan. Warna dari slidertail ditentukan dari warna kombo yang digunakan oleh mapper pada beatmap mereka.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
should be alright for me 👍

**Review by Nivalyx — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**